mlrun/api/utils/projects/follower.py

```diff
@@ -83,6 +83,7 @@ def create_project(
         projects_role: typing.Optional[mlrun.api.schemas.ProjectsRole] = None,
         leader_session: typing.Optional[str] = None,
         wait_for_completion: bool = True,
+        commit_before_get: bool = False,
```

Comment:
Why do we default to false here?
And here is the value set to true so we indeed commit before the get?